packages/cascara/src/modules/DataDate/DataDate.fixture.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import DataDate from './DataDate';
+import ModuleSandbox from '../ModuleSandbox';
+
+const DisplayDataDate = ({ isEditing, value, ...rest }) => (
+  <ModuleSandbox isEditing={isEditing}>
+    <DataDate
+      {...rest}
+      label='Date'
+      max={'2021-07-25'}
+      min={'2021-07-23'}
+      value={value}
+    />
+  </ModuleSandbox>
+);
+
+const DataDateModule = () => {
+  return (
+    <>
+      <div>
+        <h1>Date input field</h1>
+        <p>
+          This fixture is for testing the presentation of an input date field
+        </p>
+      </div>
+
+      <hr />
+
+      <div>
+        <h2>Display</h2>
+        <DisplayDataDate isEditing={false} value={'2021-07-24'} />
```

Comment:
If we have a boolean prop, we do not need to define false on it. We can safely leave off isEditing to set it as false.